runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -244,7 +252,8 @@
                 try {
                     router.addClient(new LayoutClient())
                             .addClient(new SequencerClient())
-                            .addClient(new LogUnitClient())
+                            .addClient(new LogUnitClient().setMetricRegistry(metrics != null
+                                            ? metrics : CorfuRuntime.getDefaultMetrics()))
```

Comment:
not sure why LogUnitClient is the only one that needs to maintain the MetricsRegistry in its own local field; anyway, this is legacy, not new.